src/Open3D/GUI/SceneWidget.h

```diff
@@ -64,6 +64,14 @@ class SceneWidget : public Widget {
                      const geometry::AxisAlignedBoundingBox& geometryBounds,
                      const Eigen::Vector3f& centerOfRotation);
 
+    /// Enables changing the directional light with the mouse.
+    /// SceneWidget will update the light's direction, so onDirChanged is
+    /// only needed if other things need to be updated (like a UI).
+    void SetDirectionalLight(
```

Comment:
I think, we will have bunch of `SelectSomething()` methods in this class, maybe start with directional light and rename function to `SelectDirectionalLight`?

Reply:
Ok